aptos-move/move-examples/mint_nft_v2/4-Getting-Production-Ready/output.key

```diff
@@ -0,0 +1 @@
+C6FB9CC1022DF74BE74EEC60F93D0F0C3CC4F75C92DEEC4486BDFE57FE67A011
```

Comment:
What is this?

Reply:
To be quite honest, I didn't fully understand the purpose of generating a public key in the old tutorial. I think it may have been a relic of a convoluted process to test the module, since there is no easy way to generate a public key and private key in a `[#test]` function

Reply:
> > Great job! Would this PR also update the content in `aptos.dev`? Where is the menu that leads to the tutorial of this example in `aptos.dev`?
> 
> I'm not sure how we're going to change the structure to add it, since we're going to be keeping the old one. For now, I'm going to show it next to the other one, just add a `v2` to the name:
> 
> `Mint NFTs (v2) with the Aptos CLI`
> 
> the url would be https://aptos.dev/guides/nfts/mint-nft-v2-cli
> 
> What do you guys think? @movekevin

That sounds fine on paper. I'd just make the change and we can then look at preview to see if it makes sense